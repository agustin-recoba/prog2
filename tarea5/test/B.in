apilar 5
apilar 6
estaVaciaPila
encolar 2 
insertarEnAvl 10
insertarEnAvl 8
insertarEnAvl 12
insertarEnAvl 14
insertarEnAvl 16
singleton 7
unionDeConjuntos 2 3 6
desapilar
cima
frente
desencolar
izqAvl
alturaDeAvl
enOrdenAvl
avlMin 3
iteradorDeConjunto
Fin
