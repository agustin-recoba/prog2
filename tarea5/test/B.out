1>Apilado.
2>Apilado.
3>p NO está vacia.
4>Encolado.
5>Insertado.
6>Insertado.
7>Insertado.
8>Insertado.
9>Insertado.
10>Creado el singleton.
11>Union.
12>Desapilado.
13>5.
14>Binario vacío.
15>Desencolado.
16>8
17>Altura de avl: 3.
18>8 10 12 14 16 
19>
-(4,0.00)
(3,0.00)
-(2,0.00)
--(1,0.00)
20>3 6 7 
21>Fin.
